dtos/http_response.go

```diff
@@ -0,0 +1,39 @@
+package dtos
+
+type ErrorResponse struct {
+	Code    int         `json:"code"`
+	Message string      `json:"message"`
+	Detail  interface{} `json:"detail"`
+}
+
+type ResponseDto struct {
+	Message string        `json:"message"`
+	Data    interface{}   `json:"data"`
+	Error   ErrorResponse `json:"error"`
+}
+
+type CreateUpdateFormResponseDto struct {
```

Comment:
where is UpdatedBy

Reply:
done

Reply:
- @iamitprakash why do we need `updatedBy` here, i do not see it being required in the response dto, as UI will not be consuming this value.